### ISSUE TITLE
Issue with postInstallation when using rolenames with dash

### ISSUE THREAD

**Issue by devops-42** [bitnami/bitnami-docker-postgresql#82]
**Description**

When deploying the `bitnami-docker-postgresql` image with usernames containing a dash "-" the `postInstallation` process fails.

**Steps to reproduce the issue:**

1. Deploy image either by helm or manually
2. Provide environment variable `POSTGRESQL_USERNAME` with value `foo-bar`
3. Verify log output

**Describe the results you received:**

```
postgre INFO  ==> Deploying PostgreSQL from scratch...
postgre INFO  ==> Creating database foo-bar...
postgre INFO  ==> Creating user foo-bar...
Error executing 'postInstallation': ERROR:  syntax error at or near "-"
LINE 1: CREATE ROLE foo-bar WITH LOGIN CREATEDB PASSWO...
                       ^
```

The user `foo-bar` is created and has access to its database.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Client:
 Version:         1.13.1
 API version:     1.26
 Package version: docker-1.13.1-63.git94f4240.el7.centos.x86_64
 Go version:      go1.9.4
 Git commit:      94f4240/1.13.1
 Built:           Fri May 18 15:44:33 2018
 OS/Arch:         linux/amd64

Server:
 Version:         1.13.1
 API version:     1.26 (minimum version 1.12)
 Package version: docker-1.13.1-63.git94f4240.el7.centos.x86_64
 Go version:      go1.9.4
 Git commit:      94f4240/1.13.1
 Built:           Fri May 18 15:44:33 2018
 OS/Arch:         linux/amd64
 Experimental:    false
```
**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

Deployment is done via MiniShift.

Thanks for your help!

**Comment by juan131** [comment]
Hi @devops-42 

You're right! There's an issue since it's not double-quoting the username so it can't use names with dashes such as **foo-bar**. It's running the command:

```
CREATE ROLE foo-bar WITH LOGIN CREATEDB PASSWORD 'XXXX'
```

It should be running the command:

```
CREATE ROLE "foo-bar" WITH LOGIN CREATEDB PASSWORD 'XXXX'
```

I will open a task to address this issue asap. Thanks for reporting it.

**Comment by juan131** [comment]
Hi @devops-42 

I just released the version **10.4.0-debian-9-r33** of this Docker Image. You should be able to use rolenames with dashes using this latest version.

Could you please give it a try and let us know if that fixes the issue?

**Comment by devops-42** [comment]
Hi @juan131 ,

works like a charm! The user has been created with all necessary permissions.

Thanks for your help!
